handlers/svg-open-street-map/osm-svg.py

```diff
@@ -0,0 +1,216 @@
+        lon = data["bounds"]["longitude"]
+        lon_min = lon["min"]
+        lat_min = lat["min"]
+        lon_max = lon["max"]
+        lat_max = lat["max"]
+        # Scale the lon/lat units to svg pixels equivalent.
+        scaled_longitude = dimensions[0] / (lon_max - lon_min)
+        scaled_latitude = dimensions[1] / (lat_max - lat_min)
+        bounds = [[lon_min, lat_min],
+                  [lon_min, lat_max],
+                  [lon_max, lat_max],
+                  [lon_max, lat_min],
+                  [lon_min, lat_min]]
+        # Draw bounding box for the streets
+        for index in range(len(bounds) - 1):
+            p = draw.Path(stroke="orange", stroke_width=6, fill='none')
+            p.M(scaled_longitude * (bounds[index][0] - lon_min),
+                scaled_latitude * (bounds[index][1] - lat_min))
+            p.L(scaled_longitude * (bounds[index + 1][0] - lon_min),
+                scaled_latitude * (bounds[index + 1][1] - lat_min))
+            svg.append(p)
+
+        colors = [
+            "red",
+            "blue",
+            "springgreen",
+            "deeppink",
+            "orange",
+            "purple",
+            "cyan",
+            "coral",
+            "teal",
+            "indigo",
+            "lime",
+            "chocolate",
+            "magenta",
+            "crimson",
+            "deepskyblue",
+            "greenyellow",
+            "gold",
+            "green",
+            "aqua",
+            "navy",
+            "royalblue",
+            "forestgreen",
+            "dodgerblue"
+        ]
+        # Draw the streets with svg.
+        for street in range(len(streets)):
+            color = street
+            if street > len(colors) - 1:
+                color = len(colors) - 1
+            p = draw.Path(
+                stroke=colors[color],
+                stroke_width=1.5,
+                fill='none')
+            node_coordinates = [[node["lon"], node["lat"]]
+                                for node in streets[street]["nodes"]]
+            for index in range(len(node_coordinates) - 1):
+                p.M(scaled_longitude *
+                    (node_coordinates[index][0] -
+                     lon_min), scaled_latitude *
+                    (node_coordinates[index][1] -
+                     lat_min))
+                p.L(scaled_longitude *
+                    (node_coordinates[index +
+                                      1][0] -
+                     lon_min), scaled_latitude *
+                    (node_coordinates[index +
+                                      1][1] -
+                     lat_min))
+                svg.append(p)
+            if "street_name" in streets[street]:
+                svg_layers.append(
+                    {"label": streets[street]["street_name"],
+                        "svg": svg.asDataUri()})
+            else:
+                svg_layers.append(
+                    {"label": str(streets[street]["street_id"]),
```

Comment:
Unsure how useful this will be in practice, please post an SVG in this format _as an image_.

Reply:
![image](https://user-images.githubusercontent.com/25706846/222882131-c6bcb683-0f1e-42ae-a038-a4e7fdc70997.png)

Reply:
![image](https://user-images.githubusercontent.com/25706846/222882183-de345eed-fcc3-4895-a3f5-6a0b79e68dd2.png)

Reply:
![image](https://user-images.githubusercontent.com/25706846/222882228-ab60b87e-a1e9-49b8-a6c7-187d36203125.png)

Reply:
![image](https://user-images.githubusercontent.com/25706846/222882254-da005357-a9e6-43f4-80dc-492731f731be.png)

Reply:
![image](https://user-images.githubusercontent.com/25706846/222882305-678d679b-42bb-4bc3-84bd-53d0e1cb6772.png)